pkg/kn/commands/source/container/create_test.go

```diff
@@ -60,3 +60,22 @@ func TestNoSinkError(t *testing.T) {
 	_, err := executeContainerSourceCommand(containerClient, nil, "create", "testsource", "--image", "docker.io/test/testimg")
 	assert.ErrorContains(t, err, "required flag(s)", "sink", "not set")
 }
+
+func TestContainerCreateErrorForNoArgs(t *testing.T) {
+	containerClient := v1.NewMockKnContainerSourceClient(t, "mynamespace")
+	argMissingMsg := "requires the name of the source to create as single argument"
+	_, err := executeContainerSourceCommand(containerClient, nil, "create", "--sink", "ksvc:testsvc", "--image", "docker.io/test/testimg")
+	assert.Error(t, err, argMissingMsg)
+}
+
+func TestContainerCreatePSError(t *testing.T) {
+	testsvc := &servingv1.Service{
+		TypeMeta:   metav1.TypeMeta{Kind: "Service", APIVersion: "serving.knative.dev/v1"},
+		ObjectMeta: metav1.ObjectMeta{Name: "testsvc", Namespace: "default"},
+	}
+	dynamicClient := dynamicfake.CreateFakeKnDynamicClient("default", testsvc)
+	containerClient := v1.NewMockKnContainerSourceClient(t)
+
+	_, err := executeContainerSourceCommand(containerClient, dynamicClient, "create", "testsource", "--sink", "ksvc:testsvc", "--image", "docker.io/test/testimg", "--mount", "123456")
+	assert.Error(t, err, "cannot create ContainerSource 'testsource' in namespace 'default' because: Invalid --mount: argument requires a value that contains the \"=\" character; got \"123456\"")
```

Comment:
You could go with `assert.Assert(t, util.ContainsAll(out, ....)`, but it's minor nit really. I'm sure @rhuss would also notice though. :)

Reply:
oh instead of comparing the full error message, just check for key words ?

Reply:
Yes exactly, so we don't have to maintain the whole string that might be prone to changes.

Reply:
yes

Reply:
have made the changes.